converters/fromZigbee.js

```diff
@@ -5007,6 +5007,33 @@ const converters = {
             }
         },
     },
+    SAGE206612_state: {
+        cluster: 'genOnOff',
+        type: 'commandOn',
+        convert: (model, msg, publish, options, meta) => {
+            const deviceId = msg.endpoint.deviceIeeeAddress;
+            const timeout = 28;
+
+            if (!store[deviceId]) {
+                store[deviceId] = [];
+            }
+
+            const timer = setTimeout(() => {
```

Comment:
Could you explain/add a comment why this timer stuff is here? Why not just return `{"action": "on"}`

Reply:
Sure, it's for 2 reason:
1. It's to avoid multiple notification in case of the bell is pressed multiple in a short range 
2. 25 seconds after the bell is detected, the device sends a second message which would be interpreted as a change of state. Unfortunately, the body of sent messages is always the same and therefore it is not possible to distinguish them except with a timer

Alternative solutions to solving this type of problem are obviously welcome!